x-pack/plugins/security/public/authentication/login/components/login_form/login_form.tsx

```diff
@@ -137,128 +230,146 @@ export class LoginForm extends Component<Props, State> {
+          <button
+            key={provider.name}
+            disabled={!this.isLoadingState(LoadingStateType.None)}
+            onClick={() =>
+              provider.usesLoginForm
+                ? this.onPageModeChange(PageMode.Form)
+                : this.loginWithSelector(provider.type, provider.name)
+            }
+            className={`lgnCard ${
+              this.isLoadingState(LoadingStateType.Selector, provider.name)
+                ? 'lgnCard-isLoading'
+                : ''
+            }`}
+          >
+            <EuiFlexGroup alignItems="center" gutterSize="m" responsive={false}>
+              <EuiFlexItem grow={false}>
+                <EuiIcon size="xl" type={provider.icon ? provider.icon : 'empty'} />
+              </EuiFlexItem>
+              <EuiFlexItem>
+                <EuiTitle size="xs" className="lgnCard__title">
+                  <p>
+                    {provider.description ?? (
+                      <FormattedMessage
+                        id="xpack.security.loginPage.loginProviderDescription"
+                        defaultMessage="Log in with {providerType}/{providerName}"
+                        values={{
+                          providerType: provider.type,
+                          providerName: provider.name,
+                        }}
+                      />
+                    )}
+                  </p>
+                </EuiTitle>
+                {provider.hint ? <p className="lgnCard__hint">{provider.hint}</p> : null}
+              </EuiFlexItem>
+              {this.isLoadingState(LoadingStateType.Selector, provider.name) ? (
+                <EuiFlexItem grow={false}>
+                  <EuiLoadingSpinner size="m" />
+                </EuiFlexItem>
+              ) : null}
+            </EuiFlexGroup>
+          </button>
+        ))}
+      </EuiPanel>
+    );
+  };
 
+  private renderLoginHelp = () => {
     return (
-      <Fragment>
-        <EuiText size="s">
-          <ReactMarkdown>{this.props.loginAssistanceMessage}</ReactMarkdown>
+      <EuiPanel data-test-subj="loginHelp">
+        <EuiText>
+          <ReactMarkdown>{this.props.loginHelp || ''}</ReactMarkdown>
         </EuiText>
-      </Fragment>
+      </EuiPanel>
     );
   };
 
-  private renderMessage = () => {
-    const { message } = this.state;
-    if (message.type === MessageType.Danger) {
+  private renderPageModeSwitchLink = () => {
+    if (this.state.mode === PageMode.LoginHelp) {
       return (
         <Fragment>
-          <EuiCallOut
-            size="s"
-            color="danger"
-            data-test-subj="loginErrorMessage"
-            title={message.content}
-            role="alert"
-          />
-          <EuiSpacer size="l" />
+          <EuiSpacer />
+          <EuiText size="xs" className="eui-textCenter">
+            <EuiLink
+              data-test-subj="loginBackToLoginLink"
+              style={{ fontWeight: 'bold' }}
```

Comment:
@snide this link to return back to the login flow is `bold`, but the link to get to this screen is not bolded. Is that intentional?

![image](https://user-images.githubusercontent.com/3493255/80370285-cfed6880-885d-11ea-8888-26ca6faa35b1.png)


![image](https://user-images.githubusercontent.com/3493255/80370328-e5629280-885d-11ea-9fac-e01de0230f84.png)

Reply:
I though it was, but thanks for double checking :slightly_smiling_face:

Reply:
The font-weight style tag should be removed.